tests/foreman/cli/test_repository.py

```diff
@@ -66,6 +66,34 @@ def _make_repository(self, options=None):
 
         return make_repository(options)
 
+    def test_bugzilla_1189289(self):
+        """@Test: Check if repository docker-upstream-name is shown
+        in repository info
+
+        @Feature: Repository
+
+        @Assert: repository info command returns upstream-repository-name
+        value
+
+        """
+        new_repo = self._make_repository({
+            u'name': gen_string('alpha', 10),
```

Comment:
not a blocker, but for future reference you don't have to explicitly pass a length of 10 to gen_string because it is the default value for all the available string options.